core/src/test/groovy/com/novoda/gradle/release/TestConfiguration.groovy

```diff
@@ -73,9 +81,21 @@ class TestBintrayRelease {
         Node packagingNode = nodes[5]
         assertThat(packagingNode.name().localPart).isEqualTo "packaging"
         assertThat(packagingNode.value()[0]).isEqualTo "aar"
+    }
+
+    @Test
+    public void testBintrayUploadTask() {
+        Task task = project.getTasks().findByPath(":bintrayUpload")
+        assertThat(task).isNotNull()
+
+        BintrayUploadTask uploadTask = task as BintrayUploadTask
+        uploadTask.user = "user"
+        uploadTask.apiKey = "apiKey"
 
-        // uploadTask verification here
+        uploadTask.repoName = "maven"
```

Comment:
`repoName` and `packageName` are usually set after all projects have been evaluated. This callback (https://github.com/bintray/gradle-bintray-plugin/blob/master/src/main/groovy/com/jfrog/bintray/gradle/BintrayPlugin.groovy#L30) is not executed during test.